### PR TITLE
docs: add L7 leafelt plugin 

### PR DESCRIPTION
**Pull request by lzxue**
Enhance Leaflet Large Scale Geospatial data visualization
L7 Leaflet: leaflet-map

[L7](https://github.com/antvis/l7) Large-scale WebGL-powered Geospatial data visualization analysis engine.

### CODE REVIEW

**Comment by lzxue** [comment]
can click Approve and run workflow . @mourner

File: docs/_plugins/dataviz/leaflet-l7.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: '@antv/l7-leaflet'
+category: dataviz
+repo: https://github.com/antvis/L7-Leaflet
+author: lzxue
+author-url: https://github.com/lzxue
+demo: http://l7-leaflet.antv.vision/leaflet_l7_layer
+compatible-v0:
```
Comment (jonkoops) [inline docs/_plugins/dataviz/leaflet-l7.md:8]: @Falke-Design does this need to be set explicitly?

File: docs/_plugins/dataviz/leaflet-l7.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: '@antv/l7-leaflet'
+category: dataviz
+repo: https://github.com/antvis/L7-Leaflet
+author: lzxue
+author-url: https://github.com/lzxue
+demo: http://l7-leaflet.antv.vision/leaflet_l7_layer
```
Comment (Falke-Design) [inline docs/_plugins/dataviz/leaflet-l7.md:7]:
```suggestion
demo: https://l7-leaflet.antv.vision/leaflet_l7_layer
```

File: docs/_plugins/dataviz/leaflet-l7.md (hunk 1 above)
Comment (Falke-Design) [reply docs/_plugins/dataviz/leaflet-l7.md:8]:
```suggestion
compatible-v0: false
```

File: docs/_plugins/dataviz/leaflet-l7.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: '@antv/l7-leaflet'
```
Comment (Falke-Design) [inline docs/_plugins/dataviz/leaflet-l7.md:2]:
```suggestion
name: @antv/l7-leaflet
```

File: docs/_plugins/dataviz/leaflet-l7.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+category: dataviz
+repo: https://github.com/antvis/L7-Leaflet
+author: lzxue
+author-url: https://github.com/lzxue
+demo: http://l7-leaflet.antv.vision/leaflet_l7_layer
+compatible-v0:
+compatible-v1: true
+---
+use [L7](https://github.com/antvis/l7) Large-scale WebGL-powered Geospatial data visualization analysis engine, Enhance Leaflet Large Scale Geospatial data visualization
```
Comment (Falke-Design) [inline docs/_plugins/dataviz/leaflet-l7.md:11]:
```suggestion
Use [L7](https://github.com/antvis/l7) Large-scale WebGL-powered Geospatial data visualization analysis engine, Enhance Leaflet Large Scale Geospatial data visualization
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]
small fixes

File: docs/_plugins/dataviz/leaflet-l7.md (hunk 1 above)
Comment (Falke-Design) [reply docs/_plugins/dataviz/leaflet-l7.md:8]: No not explicitly but I often make a commit and overwrite it with `false`.

BTW: should we introduce then compatible-v2?

**Review by Falke-Design — APPROVED** [review]

File: docs/_plugins/dataviz/leaflet-l7.md (hunk 1 above)
Comment (jonkoops) [reply docs/_plugins/dataviz/leaflet-l7.md]: I think we can wait with `v2` until we stabilized things.

**Review by jonkoops — APPROVED** [review]

**Pull request merged**